plugins/UM3NetworkPrinting/src/Cloud/CloudOutputDevice.py

```diff
@@ -256,7 +256,16 @@ def _onPrintUploadCompleted(self, response: CloudPrintResponse) -> None:
         """
         self._uploaded_print_job = self._pre_upload_print_job
         self._progress.hide()
-        PrintJobUploadSuccessMessage().show()
+        message = PrintJobUploadSuccessMessage()
+        message.addAction("monitor print",
+                          name=I18N_CATALOG.i18nc("@action:button", "Monitor Print"),
+                          icon="",
+                          description="Remove the plugin",
```

Comment:
This was copied from a similar message in the PluginRegistry, but the description is not applicable here. What do you think of this?

```suggestion
                          description=I18N_CATALOG.i18nc("@action:tooltip", "Track the print in Ultimaker Digital Factory"),
```